src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
+// reducer
+const initialState = [{ status: 'UNDER CONSTRUCTION' }];
```

Comment:
- [ ] Kindly set the `intialState` with an empty array as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Set%20the%20initial%20state%20to%20be%20an%20empty%20array%20of%20categories.).